python/sglang/srt/managers/template_manager.py

```diff
@@ -129,10 +129,12 @@ def load_chat_template(
                         f"Using default HuggingFace chat template with detected content format: {self._jinja_template_content_format}"
                     )
                     return
-
-            # Default to string content format if no template was found
-            self._jinja_template_content_format = "string"
-            logger.info("No chat template found, defaulting to 'string' content format")
+                else:
+                    # Default to string content format if no template was found
+                    self._jinja_template_content_format = "string"
+                    logger.info(
+                        "No chat template found, defaulting to 'string' content format"
+                    )
 
         # Detect reasoning pattern from chat template
         if tokenizer_manager.tokenizer:
```

Comment:
Hi @JustinTong0323 , I am not familiar with the logic below. Can you help confirm that it is expected that we should set `_force_reasoning` in all cases unless we can resolve a hf_template? Thanks!